src/onegov/org/cronjobs.py

```diff
@@ -826,29 +827,34 @@ def delete_unconfirmed_newsletter_subscriptions(request: OrgRequest) -> None:
 
 
 def get_news_for_push_notification(session: Session) -> Query[News]:
+    # Use UTC time for database comparisons since publication_start is stored
+    # in UTC
     now = utcnow()
-    ten_minutes_ago = now - timedelta(minutes=10)
 
     # Get all news items that should trigger push notifications
     query = session.query(News)
-    query = query.filter(News.published == True)
+    query = query.filter(News.published.is_(True))
     query = query.filter(News.publication_start <= now)
+    query = query.filter(News.publication_start.isnot(None))
```

Comment:
```suggestion
```

I'm pretty sure this condition is redundant, since the comparison above already gets rid of rows with a NULL value.